cmd/dogstatsd/subcommands/start/command.go

```diff
@@ -114,19 +114,16 @@ func RunDogstatsdFct(cliParams *CLIParams, defaultConfPath string, defaultLogFil
 		// workloadmeta setup
 		collectors.GetCatalog(),
 		fx.Provide(func(config config.Component) workloadmeta.Params {
-			var catalog workloadmeta.AgentType
-			if flavor.GetFlavor() == flavor.ClusterAgent {
-				catalog = workloadmeta.ClusterAgent
-			} else {
-				catalog = workloadmeta.NodeAgent
-			}
+			catalog := workloadmeta.NodeAgent
```

Comment:
Is it expected catalog is never set to `workloadmeta.ClusterAgent`?

Reply:
Yes, for `dogstatsd` I think this is expected.